### ISSUE TITLE
Parsing Last Name, First Name With Trailing Comma 

### ISSUE THREAD

**Issue by sshaw** [chorn/nameable#6]
```
irb [2.1.5] (nameable)$ Nameable.parse("Shaw, Skye").first
=> "Skye"
irb [2.1.5] (nameable)$ Nameable.parse("Shaw, Skye,").first
=> "Skye,"
```

It stems from [this line](https://github.com/chorn/nameable/blob/917c134cfcbf669e4b667a5d7ef1f88cd6442bb0/lib/nameable/latin.rb#L121). I would fix it but I'm not sure what this line trying to do. Nothing in the specs seems to test `Last Name, First Name`. No clues there. 

Calling `#to_s` on the parsed name normalizes it into a format that can be parsed: 

```
irb [2.1.5] (nameable)$ Nameable.parse("Shaw, Skye,").to_s
=> "Skye, Shaw"
```

**Comment by chorn** [comment]
This seems like a bug to me.

**Comment by chorn** [comment]
This is fixed in v1.1.3:

https://github.com/chorn/nameable/releases/tag/v1.1.3
